README.md

```diff
@@ -138,6 +138,9 @@ Or, in case of just a `from` with the default destination, you can also use a `{
       from: 'src/'
       to: 'dest/[name].[hash].[ext]',
       toType: 'template'
+      // Optional: provide a regExp to extract additional placeholders
```

Comment:
Please make a new 'section'  for this option, or does it only work with `toType: 'template'` ?

Reply:
Also needs to be added to `options` table please

Reply:
It will only work with the template `toType`. Will update the table.